planet/src/main/webapp/js/placesItinerary.js

```diff
@@ -16,6 +16,8 @@
 window.openAddPlaceForm = openAddPlaceForm;
 window.closeAddPlaceForm = closeAddPlaceForm;
 
+const database = firebase.database();
```

Comment:
nit: Weren't you able to use the global variable in the authentication.js file?

Reply:
See comment above